.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Provide a general description of the bug, in plain text. Use the dedicated fields below for code snippets.
+      placeholder: Description of the bug
+    validations:
+      required: true
+  - type: textarea
+    id: java-input
+    attributes:
+      label: Source code you are trying to analyze/transform
+      description: Provide a minimal example of source code you are trying to analyze/transform that causes the bug to appear.
```

Comment:
Is this also automatically formatted? (I think so, but do we want to explicitly specify it here as well?)

Reply:
Yes, it's the "render" attribute. I've just forgotten to add the little text snippet about the formatting.

Reply:
```suggestion
      description: Provide a minimal example of source code you are trying to analyze/transform that causes the bug to appear. This will be automatically formatted, so no need for backticks.
```